tests/pytests/test_raw_docid_encoding.py

```diff
@@ -0,0 +1,125 @@
+"""
+Tests for RAW_DOCID_ENCODING configuration.
+
+These tests should be run with RAW_DOCID_ENCODING=true module argument to verify
+that the raw DocID encoding (4-byte fixed) works correctly with TAG indexes and GC.
+"""
+
+from common import skip, config_cmd, waitForIndex, forceInvokeGC
+
+
+@skip(cluster=True)
+def testRawDocIdEncodingConfig(env):
+    """Verify RAW_DOCID_ENCODING is enabled.
+
+    This test skips if RAW_DOCID_ENCODING is not set to 'true', since these tests
+    are meant to be run with the RAW_DOCID_ENCODING=true module argument.
+    """
+    raw_encoding = env.cmd(config_cmd(), 'GET', 'RAW_DOCID_ENCODING')
+    # Config returns [['RAW_DOCID_ENCODING', 'true']] format
+    if raw_encoding[0][1] != 'true':
+        env.skip()
+    env.assertEqual(raw_encoding[0][1], 'true')
+
+
+@skip(cluster=True)
+def testTagIndexWithRawDocIdEncoding(env):
```

Comment:
Set up an environment with raw docIDs encoding, instead of expecting it
```suggestion
def testTagIndexWithRawDocIdEncoding():
    env = Env(moduleArgs='RAW_DOCID_ENCODING true')
```